src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -1807,12 +1846,17 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         const tag = this.props.project.tags.find((tag) => tag.name === tagName);
 
         let regionCategory;
-        if (labelType) {
-            regionCategory = labelType;
-        } else if (tag.type === FieldType.SelectionMark) {
-            regionCategory = FeatureCategory.Checkbox;
-        } else {
-            regionCategory = FeatureCategory.Text;
+        try {
+            if (labelType) {
+                regionCategory = labelType;
+            } else if (tag.type === FieldType.SelectionMark) {
+                regionCategory = FeatureCategory.Checkbox;
+            } else {
+                regionCategory = FeatureCategory.Text;
+            }
+        }
+        catch (err) {
+            console.error(err);
```

Comment:
there should be no error here. We fixed the tag type undefined error with #557

Reply:
so you don't need a try catch

Reply:
removed and push to github